Objective-C/CBLBlob.mm

```diff
@@ -147,9 +151,10 @@ - (NSDictionary*) properties {
         return _properties;
     } else {
         // New blob:
-        return $dict({@"digest", _digest},
-                     {@"length", (_length ? @(_length) : nil)},
-                     {@"content_type", _contentType});
+        return $dict({kTypeMetaProperty, kBlobType},
```

Comment:
To simplify, we don't keep the _properties and we always return a new dict.

Reply:
This method is not thread safe as _length could be changed. Can you also synchronize this method?